charts/garage-signer/templates/secret-store.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if .Values.useExternalSecret }}
+apiVersion: "external-secrets.io/v1beta1"
+kind: SecretStore
+metadata:
+  name: {{ $.Chart.Name }}-secretsmanager
+  namespace: {{ $.Chart.Name }}
```

Comment:
some namespaces use `{{ $.Chart.Name }}` and others use `{{ $.Values.namespace }}`.
maybe we need to unify namespace